test/index.ts

```diff
@@ -108,4 +108,12 @@ describe('decodeResumeToken', function() {
       documentKey: null
     });
   });
+
+  // See PM-1950
+  it.only('can decode new resume tokens', async() => {
+    const decoded = decodeResumeToken('826273BCD2000000012B022C0100296E5A1004808BBE9298B0403286C29EA48C4E162846645F696400646273BCD27257D30393BB73F70004');
+    console.log(decoded);
+    assert.deepStrictEqual(bson.EJSON.serialize(decoded), {
```

Comment:
```
  1) decodeResumeToken
       can decode new resume tokens:
     BSONTypeError: Unrecognized or invalid _bsontype: UUID
      at new BSONTypeError (node_modules/bson/src/error.ts:16:5)
      at serializeDocument (node_modules/bson/src/extended_json.ts:304:15)
      at serializeValue (node_modules/bson/src/extended_json.ts:243:58)
      at serializeDocument (node_modules/bson/src/extended_json.ts:287:22)
      at serializeValue (node_modules/bson/src/extended_json.ts:243:58)
      at stringify (node_modules/bson/src/extended_json.ts:426:17)
      at Object.serialize (node_modules/bson/src/extended_json.ts:438:23)
      at Context.<anonymous> (test/index.ts:116:39)
```

Reply:
https://github.com/mongodb/js-bson/blob/b86cabf76856e64c6a1f43f900711795a76cd66e/src/extended_json.ts#L247-L274

Maybe this? https://github.com/mongodb/js-bson/pull/465